distribution/sender/routes/sender-activemq-consumer.xml

```diff
@@ -0,0 +1,93 @@
+<routes xmlns="http://camel.apache.org/schema/spring">
+    <route id="sender-activemq-consumer" errorHandlerRef="shutdownErrorHandler">
+        <from uri="activemq:openmrs.sync.{{db-sync.senderId}}?connectionFactory=activeMqConnFactory&amp;acknowledgementModeName=CLIENT_ACKNOWLEDGE&amp;messageListenerContainerFactory=customMessageListenerContainerFactory&amp;asyncStartListener=true" />
+
+        <log message="Received message -> ${body}" />
+        
+        <setProperty name="requestUuid">
+            <jsonpath suppressExceptions="true">$.requestUuid</jsonpath>
+        </setProperty>
+        <setProperty name="messageUuid">
+            <jsonpath suppressExceptions="true">$.messageUuid</jsonpath>
+        </setProperty>
+        
+        <choice>
+            <when>
+                <simple>${exchangeProperty.requestUuid} != null</simple>
```

Comment:
I think here you should instead check if operation.operation is not set to **r** on the incoming message.